### PR TITLE
[PHI]Unify  fluid kernel (Part3)

### PR DESCRIPTION
**Pull request by YuanRisheng**
### PR types

Others
### PR changes

Others
### Describe

Pcard-67000
将以下Fluid op注册到phi中：
     push_dense
     pyramid_hash
     quantize_linear
     random_crop
     random_routing
     rank_attention
     read_file

### CODE REVIEW

File: paddle/fluid/operators/rank_attention_op.cu [hunk 1]
```diff
@@ -243,10 +243,16 @@ class RankAttentionGradOpCUDAKernel : public framework::OpKernel<T> {
 
 namespace ops = paddle::operators;
 using GPUCtx = phi::GPUContext;
-REGISTER_OP_CUDA_KERNEL(rank_attention,
```
Comment (jiahy0825) [inline paddle/fluid/operators/rank_attention_op.cu:246]: Maybe `line 245` could also be deleted
`using GPUCtx = phi::GPUContext;`

**Review by jiahy0825 — APPROVED** [review]
LGTM

File: paddle/fluid/operators/rank_attention_op.cu (hunk 1 above)
Comment (YuanRisheng) [reply paddle/fluid/operators/rank_attention_op.cu:246]: 3q, I will do it in next pr.

**Review by huangjiyi — APPROVED** [review]

File: paddle/fluid/operators/quantize_linear_op.cu [hunk 2]
```diff
@@ -123,12 +123,18 @@ template struct ChannelDequantizeFunctorV2<phi::GPUContext, double>;
 
 namespace ops = paddle::operators;
 using CUDA = phi::GPUContext;
-REGISTER_OP_CUDA_KERNEL(dequantize_linear,
```
Comment (huangjiyi) [inline paddle/fluid/operators/quantize_linear_op.cu:126]: 这个 `using CUDA` 应该可以删了

**Review by heavyrain-lzy — APPROVED** [review]

File: paddle/fluid/operators/quantize_linear_op.cu (hunk 2 above)
Comment (YuanRisheng) [reply paddle/fluid/operators/quantize_linear_op.cu:126]: 3q, I will do it in next pr.

**Pull request merged**